.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('pa11y-lint-config/eslint/es6');
+module.exports = require('pa11y-lint-config/eslint/es2017');
```

Comment:
What's the reason for using `es2017` here? I'm not aware of any 2017 functionality

Reply:
Aligned with https://github.com/pa11y/pa11y/blob/master/.eslintrc.js